script.js

```diff
@@ -6,4 +9,317 @@ function openNav(){
+                    {
+                        var str = cookieList[i];
+                        var checkerName = str.substring(str.lastIndexOf('/')+1,str.indexOf('='));
+                        if (checkerName == newElementName)
+                        {
+                            elementAlreadyStored = true;
+                            break;
+                        }
+                    }
+                }
+                
+                if (!elementAlreadyStored)
+                {
+                    $("#elementConfirmationBox").text(" ");
+                    document.cookie = cookieName + "=" + cookieValue + 
+                    "; expires=Wed, 1 Jan 2038 12:00:00 UTC; path=/";
+                    //doc.cookie --> /season_config/season/elementName=elementValue;...
+
+                    var newTRHtml = "<tr><td>" + newElementName + 
+                    "</td><td>" + newElementPoints + "</td></tr>";
+                    
+                    $("#scoringTable").append(newTRHtml);
+                    $("#elementConfirmationBox").html('<i>Element "' + newElementName + '" added.<\i>');
+
+                    $("#elementKey").val("");
+                    $("#elementPoints").val("");
+                } else
+                {
+                    $("#elementConfirmationBox").html('<i>Element "' + newElementName + '" already exists.<\i>');
+                }
+            } else
+            {
+                $("#elementConfirmationBox").html("<i>Please check a season.<\i>");
+            }
+        } else
+        {
+            $('#elementConfirmationBox').html("<i>Please fill in both fields.<\i>");
+        }      
+    });
+
+
+    //start of questionnaire js code
+    $("#pitScoutingSubmit").click(function(){
+        var allQuestions = $("#pitScoutingTextBox").val().split('\n');
+
+        var currSeason = getCookie("currCheckedSeason");
+        console.log(allQuestions);
+        //console.log(currSeason);
+        //console.log(currSeason);
+        //console.log(questionsArr);
+      
+        if (currSeason === "null" || currSeason ===" ")
+        {
+            $("#pitScoutConfirmationBox").html("<i>Please check a season.<\i>");
+        } else
+        {
+            for (var i = 0; i < allQuestions.length;i++)
+            {
+                // pit scouting cookies should be in the format:
+                // (currCheckedSeason) pitScout Q(i): question; expiry date...
+                var question = allQuestions[i].toString();
+                var cookieName = currSeason + " pitQuestions";
+                if (question != "")
+                {
+                    if (!getCookie(cookieName).includes(question))
+                    {
+                        //cookie --> {season} questions = {q1},{q2},{q3}
+                        document.cookie = cookieName + '=' + getCookie(cookieName) + question + ',; expires=Wed, 1 Jan 2038 12:00:00 UTC; path=/';
```

Comment:
A) use setCookie

B) - don't seperate on comma's. What if the user wants to use that in the question?
`How many cargo, if any, can you get in auto?` - this will get parsed back out at three separate.
separate on a character the user can't type. You can use `\n` (if you're trimming anyways), or a Unicode character (although, careful with this, I setup all my machines to let me type a set of Unicode characters for nice clean communication - like 15° or 220 Ω :) )

Reply:
A) ok
B) I didn't think of that, I ended up using Ω because most users won't be able to type that